src/main/scala/ch/epfl/bluebrain/nexus/iam/marshallers/instances.scala

```diff
@@ -129,6 +136,20 @@ object instances extends FailFastCirceSupport {
       }
     }
 
+  implicit val permissionsStatusCode: RejectionStatusCode[PermissionsRejection] =
+    new RejectionStatusCode[PermissionsRejection] {
+      override def apply(rej: PermissionsRejection): StatusCode = rej match {
+        case CannotSubtractEmptyCollection          => StatusCodes.BadRequest
+        case _: CannotSubtractFromMinimumCollection => StatusCodes.BadRequest
+        case CannotSubtractFromEmptyCollection      => StatusCodes.BadRequest
+        case _: CannotSubtractUndefinedPermissions  => StatusCodes.BadRequest
+        case CannotAppendEmptyCollection            => StatusCodes.BadRequest
+        case CannotReplaceWithEmptyCollection       => StatusCodes.BadRequest
+        case CannotDeleteMinimumCollection          => StatusCodes.BadRequest
+        case _: IncorrectRev                        => StatusCodes.BadRequest
```

Comment:
case _: IncorrectRev                        => StatusCodes.Conflict

Why all those cases `CannotReplaceWithEmptyCollection`, `CannotAppendEmptyCollection `, `CannotSubtractFromEmptyCollection`? This is all `StatusCodes .NotFound` to me.

Reply:
they are part of the logic, fixed the incorrect rev